### PR TITLE
Parse

### PR DESCRIPTION
**Pull request by ChuWeiChang**
Completion of user library. I will look into publishing it on pypl tonight.
It is a little troublesome to modify guest.py automatically in testing stage, so I tried to make it more complicated.

In python, list and tuple is really confusing, so I only make a simple check on the two data structures. I look to make it more complete in the future by issuing a reminder.

### CODE REVIEW

File: tests/test_kalgory.py [hunk 1]
```diff
@@ -32,5 +33,20 @@ def test_execute(self, generate_bindings):
         with Store() as store:
             from tests.bindings import Root
 
+            data = {"o1": 1, "o2": 20010714.0, "o3": (123, "fefe")}
```
Comment (ZhaoTzuHsien) [inline tests/test_kalgory.py:36]: The testing data should be the following for good
```json
[1, 20010714.0, [123, "fefe"]]
```

**Comment by ZhaoTzuHsien** [comment]
I think there are some misunderstandings when I was reviewing your code.

According to #6, the payload format is a JSON array. Each of its items in the array is a parameter to the user handle function. You can review the issue #6 thoroughly to see complete implementation guidance. If you have any question about the payload format or implementation problem, feel free to ask any question.

@ChuWeiChang

**Review by ZhaoTzuHsien — CHANGES_REQUESTED** [review]
Rewrite of the implementation that follows #6 is required.

File: tests/test_kalgory.py (hunk 1 above)
Comment (ChuWeiChang) [reply tests/test_kalgory.py:36]: i understand what you trying to say, but what do you mean "for good "

**Review by ZhaoTzuHsien — APPROVED** [review]
LGTM. Can be merged now.

**Pull request merged**